### PR TITLE
Modified implementations of creating function of some clients, and fixed some bugs.

### PR DESCRIPTION
**Pull request by markyu92**
This PR contains the following improvements:
1. Modified the implement functions of some clients.
2. Fixed a bug of the value of `Endpoint` in `blockStorageV3Client`.
3. Updated some parameters of test cases in `resource_huaweicloud_cce_node_v3_test.go`.

### CODE REVIEW

File: huaweicloud/endpoints.go [hunk 1]
```diff
@@ -33,22 +33,112 @@ var allServiceCatalog = map[string]ServiceCatalog{
...
 		WithOutProjectID: true,
 	},
-
+	// ******* client for Compute start *******
 	"ecs": ServiceCatalog{
 		Name:    "ecs",
 		Version: "v1",
 	},
-	"compute": ServiceCatalog{
+	"computeV11": ServiceCatalog{
```
Comment (ShiChangkuo) [inline huaweicloud/endpoints.go:41]: suggest to add the catalog with lower-case

File: huaweicloud/endpoints.go [hunk 2]
```diff
@@ -33,22 +33,112 @@ var allServiceCatalog = map[string]ServiceCatalog{
...
+	// ******* client for storage start ******//
+	"blockStorageV2": ServiceCatalog{
+		Name:    "evs",
+		Version: "v2",
+	},
+	"blockStorageV3": ServiceCatalog{
+		Name:    "evs",
+		Version: "v3",
+	},
+	"evsV2": ServiceCatalog{
```
Comment (ShiChangkuo) [inline huaweicloud/endpoints.go:87]: can we merge it with `blockStorageV2`?

File: huaweicloud/endpoints.go [hunk 3]
```diff
@@ -33,22 +33,112 @@ var allServiceCatalog = map[string]ServiceCatalog{
...
+		Name:             "vpc",
+		Version:          "v1",
+		WithOutProjectID: true,
+	},
+	"networkV2": ServiceCatalog{
+		Name:             "vpc",
+		Version:          "v2.0",
+		WithOutProjectID: true,
+	},
+	"networkingHwV2": ServiceCatalog{
```
Comment (ShiChangkuo) [inline huaweicloud/endpoints.go:116]: can we merge it with `networkV2`?

File: huaweicloud/endpoints.go (hunk 1 above)
Comment (markyu92) [reply huaweicloud/endpoints.go:41]: okay, will do this in next patch.

File: huaweicloud/endpoints.go (hunk 2 above)
Comment (markyu92) [reply huaweicloud/endpoints.go:87]: I will merge the same kind of client in next patch, may take a moment.

**Pull request merged**